homeassistant/components/coinbase.py

```diff
@@ -0,0 +1,65 @@
+"""
+Support for Coinbase.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/coinbase/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_API_KEY
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['coinbase==2.0.6']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'coinbase'
+
+CONF_API_SECRET = 'api_secret'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+
+NETWORK = None
```

Comment:
Please don't use global variables to communicate between the component and platform. Use `hass.data[…]` instead.

Reply:
Thanks! Been away from HA for a bit, I'll take a look at this next week when I have time.  The ecobee component may need an update as well.